newrelic/resource_newrelic_app_label.go

```diff
@@ -0,0 +1,172 @@
+package newrelic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/newrelic/newrelic-client-go/pkg/apm"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+	"log"
+	"strings"
+)
+
+func resourceNewRelicAppLabel() *schema.Resource {
```

Comment:
It's a bit long but changing `App` to `Application` fits with the rest of the provider.

Reply:
Renamed as suggested.